plugins/woocommerce-admin/client/marketing/overview-multichannel/Campaigns/Campaigns.tsx

```diff
@@ -0,0 +1,170 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useState } from '@wordpress/element';
+import {
+	Card,
+	CardHeader,
+	CardBody,
+	CardFooter,
+	Flex,
+	FlexItem,
+	FlexBlock,
+} from '@wordpress/components';
+import { Icon, megaphone, cancelCircleFilled } from '@wordpress/icons';
+import {
+	Pagination,
+	Table,
+	TablePlaceholder,
+	Link,
+} from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import { CardHeaderTitle } from '~/marketing/components';
+import { useCampaigns } from './useCampaigns';
+import './Campaigns.scss';
+
+const tableCaption = __( 'Campaigns', 'woocommerce' );
+const tableHeaders = [
+	{
+		key: 'campaign',
+		label: __( 'Campaign', 'woocommerce' ),
+	},
+	{
+		key: 'cost',
+		label: __( 'Cost', 'woocommerce' ),
+		isNumeric: true,
+	},
+];
+
+const perPage = 5;
+
+export const Campaigns = () => {
```

Comment:
💅  Missed JSDOCS

Reply:
Done in [`4b87f0a` (#36735)](https://github.com/woocommerce/woocommerce/pull/36735/commits/4b87f0a4ed13d00bd8054687bdab823ba10a6a03).